app/javascript/components/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Greeting from './Greeting.jsx';
+
+const App = () => {
+  return (
+    <div>
+        <Greeting />
+    </div>
```

Comment:
- [ ] Kindly setup a route in your `App` component to lead to the `Greeting` component to follow the project requirement.
![Screenshot from 2023-12-05 21-12-50](https://github.com/AliRezaBmeDu/hello-rails-react/assets/91142432/cc512709-ffc2-42ca-a132-28530aeac92c)

Reply:
done